runtime-devices/libinput/spec

```diff
@@ -1,5 +1,4 @@
-VER=1.21.0
-REL=1
+VER=1.24.0
 SRCS="tbl::https://gitlab.freedesktop.org/libinput/libinput/-/archive/$VER/libinput-$VER.tar.gz"
```

Comment:
Please try not to use automatically packaged tarballs, use Git tags instead.

Reply:
Fixed.